src/components/slide/preview/remote-component-wrapper.scss

```diff
@@ -1,6 +1,7 @@
 .remote-component-wrapper {
   width: 100%;
   height: 100%;
+  box-shadow: 0 0 1px #aaa;
```

Comment:
I think you could consider user border here.
Otherwise consider using a shadow like in https://github.com/os2display/display-templates/blob/898d24d1ee6b63e8ff5c2caa99bfd02e02162cea/src/global-styles.css#L130